LLama.Examples/Examples/ChatSessionStripRoleName.cs

```diff
@@ -1,44 +1,44 @@
-﻿using LLama.Common;
+﻿// using LLama.Common;
 
-namespace LLama.Examples.Examples
-{
-    public class ChatSessionStripRoleName
-    {
-        public static async Task Run()
-        {
-            Console.Write("Please input your model path: ");
-            var modelPath = Console.ReadLine();
-            var prompt = File.ReadAllText("Assets/chat-with-bob.txt").Trim();
+// namespace LLama.Examples.Examples
```

Comment:
Is this example no longer relevant?

Reply:
Done in d7096547317151b172bd9150203a76b587b62e8a